pkg/kubectl/describe/versioned/describe.go

```diff
@@ -753,6 +755,17 @@ func printController(controllee metav1.Object) string {
 	return ""
 }
 
+func describePodIPInfos(pod *corev1.Pod, w PrefixWriter, space string) {
+	if nil == pod.Status.PodIPs || 0 == len(pod.Status.PodIPs) {
+		w.Write(LEVEL_0, "%sIPs:\t<none>\n", space)
```

Comment:
why print anything when it is nil?  Maybe someone from CLI can advise on best practice here

Reply:
I believe `<none>` is the convention used. an example for this is events will show up as `Events: <none>` if you described a resource that has no event